packages/core/src/App/Components/Layout/Header/menu-links.jsx

```diff
@@ -37,12 +37,17 @@ const ReportTab = () => (
 );
 
 const CashierTab = observer(() => {
-    const { client, ui } = useStore();
+    const { client, ui, traders_hub } = useStore();
     const { has_any_real_account, is_virtual } = client;
     const { toggleReadyToDepositModal, toggleNeedRealAccountForCashierModal } = ui;
     const p2p_notification_count = useP2PNotificationCount();
     const real_account_needed_for_cashier = useIsRealAccountNeededForCashier();
 
+    const { content_flag } = traders_hub;
+
+    const low_risk_diel =
+        content_flag === ContentFlag.LOW_RISK_CR_NON_EU || content_flag === ContentFlag.LOW_RISK_CR_EU;
```

Comment:
@thisyahlen-deriv We have the same in `TourGuide`, Maybe you can move it to a method in `traders_hub` itself and reuse it 🤔